kolibri/core/assets/src/views/AppError/ReportErrorModal.vue

```diff
@@ -92,12 +92,14 @@
       forumUseTips: {
         message:
           'Search the community forum to see if others encountered similar issues. If unable to find anything, paste the error details below into a new forum post so we can rectify the error in a future version of Kolibri.',
+
```

Comment:
looks like the previous change might have introduced an extra newline in the output?

Reply:
I honestly have no idea why this happened - I hadn't changed anything in the code. One possibility is that the script is not idempotent. I'll take another look.

Reply:
Hrm, it does seem to be idempotent.

Reply:
OK, narrowed down the issue - seems to have been a problem that happened only when 'updating' context that hadn't changed. So I've just stopped it doing that, which seems more appropriate anyway!